frontend/app-development/features/SettingsModalButton/SettingsModal/components/Tabs/PolicyTab/PolicyTab.tsx

```diff
@@ -57,38 +55,51 @@ const subjectData: PolicySubject[] = [
   },
 ];
 
-export const PolicyEditor = () => {
-  const { org, app } = useStudioUrlParams();
+export type PolicyTabProps = {
+  /**
+   * The policy to show
+   */
+  policy: Policy;
+  /**
+   * The org
+   */
+  org: string;
+  /**
+   * The app
+   */
+  app: string;
+};
 
-  // Get the data
-  const { data: policyData, isLoading: policyLoading } = useAppPolicyQuery(org, app);
+/**
+ * @component
+ *    Displays the tab rendering the polciy for an app
+ *
+ * @property {Policy}[policy] - The policy to show
+ * @property {string}[org] - The org
+ * @property {string}[app] - The app
+ *
+ * @returns {ReactNode} - The rendered component
+ */
+export const PolicyTab = ({ policy, org, app }: PolicyTabProps): ReactNode => {
+  const { t } = useTranslation();
 
   // Mutation function to update policy
   const { mutate: updateAppPolicyMutation } = useAppPolicyMutation(org, app);
 
   /**
    * Saves the policy to backend
    */
-  const handleSavePolicy = (policy: Policy) => {
-    updateAppPolicyMutation(policy);
+  const handleSavePolicy = (updatedPolicy: Policy) => {
+    updateAppPolicyMutation(updatedPolicy);
   };
```

Comment:
Kan du ikke bruke `updateAppPolicyMutation` direkte der `handleSavePolicy` brukes nå?

Reply:
Jo, det er også en mulighet. Det eneste er at da må jeg lage en arrow function inne i TSXen, og jeg synes det ser renere ut å ha den på utsiden. I tillegg så er det enklere for debugging. 
Men om du mener det er bedre så kan det flyttes 😄